Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -792,7 +792,7 @@ public struct RenderNodeTranslator: SemanticVisitor {
                 downloadReferences[url.description] = DownloadReference(
                     identifier: downloadIdentifier,
                     renderURL: url,
-                    sha512Checksum: nil
+                    checksum: nil
```

Comment:
Possibility for a future enhancement: 
Should the author be able to specify the checksum in the directive (similar to how developers can [specify the checksum of a binary target in SwiftPM][1])?

[1]: https://github.com/apple/swift-package-manager/blob/main/Sources/PackageDescription/Target.swift#L870-L873

Reply:
I considered that in the original design for `@CallToAction`, but ultimately dropped it in favor of the local-file option. If a renderer decides to pick up the `checksum` property in the Render JSON then it will be useful, otherwise for the moment it seems like a bit of extra work for nothing.